deploy/helm/sumologic/values.yaml

```diff
@@ -253,7 +273,7 @@ prometheus-operator:
     prometheusSpec:
       externalLabels:
         # Set this to a value to distinguish between different k8s clusters
-        cluster: kubernetes
+        cluster: *cluster_name
```

Comment:
this is called an anchor used to reference a value in YAMLs. so `*cluster_name` will look for `&cluster_name` and take whatever value `&cluster_name` has

Reply:
found out this won't work if it's overriden in `helm install`, it still gets the default values `kubernetes` instead of the user-supplied value... I have remove the anchor from values.yaml. If user needs to use a different cluster name, they need to set two things 
```
--set sumologic.clusterName="test-cluster" 
--set prometheus-operator.prometheus.prometheusSpec.externalLabels.cluster="test-cluster"
```
this is not ideal, but @frankreno do you think it's acceptable for GA?